lib/KDropdownMenu.vue

```diff
@@ -175,19 +201,27 @@
         const popoverIsOpen = popover.clientWidth > 0 && popover.clientHeight > 0;
         // set current element and its siblings
         let focusedElement = document.activeElement;
-        let sibling = focusedElement.nextElementSibling;
-        let prevSibling = focusedElement.previousElementSibling;
+        // let sibling = focusedElement.nextElementSibling;
```

Comment:
Great catch! We dont need to make these calls always. We can remove these lines too.